aws/src/index.js

```diff
@@ -349,33 +348,54 @@ class S3HttpCollector extends Collector {
   }
 }
 
-/* eslint-disable no-param-reassign */
-const processLogText = (cloudWatchLogs, tagRegexMap, arn) => {
-  // derive source
-  const keys = arn.split(':');
-  const region = keys[3];
-  const accountId = keys[4];
+function deriveRegionAndAccount(arn){
+  let region;
+  let accountId;
 
-  // default value
-  let source = accountId + '-' + region;
-
-  // derive source for lambda function or EC2 instance
+  if (arn) {
+    const keys = arn.split(':');
+    region = keys[3];
+    accountId = keys[4];
+  }
+return {region, accountId};
+}
+function deriveSource(cloudWatchLogs,source){
   const logGroup = cloudWatchLogs.logGroup;
```

Comment:
Just want to confirm: logGroup, logStream is set for the batch not in individual log level right?

Reply:
The fields `logGroup`, & `logStream` are part of the input CloudWatch Log. That is why they will show up on a log message sent to the proxy. But we do not add these tags to individual `logEvents`.